doc/source/api/general/case_reader.rst

```diff
@@ -50,3 +50,11 @@ Along with basic functionality, the case reader provides many additional feature
     >>> reader.rp_vars()
     >>> reader.config_vars()
 
+
+.. automodule:: ansys.fluent.core.filereader.case
+   :members:
+   :show-inheritance:
+   :undoc-members:
```

Comment:
Should we really be setting this one?  Documenting undocumented members does not seem like a good idea, they should be documented...

Reply:
I agree, we should not expose undocumented members but if we remove `:undoc-members:` then `settings` docs will be empty and for other code base we will loose important features on large scale.  So to avoid this we have kept this option.

Reply:
I think so.